applications/nrf5340_audio/src/bluetooth/ble_core.c

```diff
@@ -110,6 +110,41 @@ static void on_bt_ready(int err)
 
 	ble_core_le_lost_notify_enable();
 
+#if (CONFIG_NRF_21540_ACTIVE)
+
+	/* Indexes for the pins gotten from nrf21540_ek_fwd.overlay */
+	uint8_t tx_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if),
+						     gpios, 0);
+	uint8_t rx_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if),
+						     gpios, 1);
+	uint8_t pdn_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if),
+						      gpios, 2);
+	uint8_t ant_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if),
+						      gpios, 3);
+	uint8_t mode_pin = NRF_DT_GPIOS_TO_PSEL_BY_IDX(
+		DT_PATH(nrf_gpio_forwarder, nrf21540_gpio_if), gpios, 4);
+
+	struct ble_hci_vs_cp_nrf21540_pins nrf21540_pins = { .mode = mode_pin,
+							     .txen = tx_pin,
+							     .rxen = rx_pin,
+							     .antsel = ant_pin,
+							     .pdn = pdn_pin,
+							     .csn = 0xffff };
```

Comment:
We might need to add a comment in here for showing that "set .csn to 0xffff because we are not using SPI to control nRF21540"